noah_frontend/package.json

```diff
@@ -25,6 +25,7 @@
     "redux": "^4.0.0",
     "redux-devtools-extension": "^2.13.5",
     "redux-logger": "^3.0.6",
-    "redux-thunk": "^2.3.0"
+    "redux-thunk": "^2.3.0",
+    "styled-components": "^3.4.5"
```

Comment:
I don't think you need to include `styled-components` since `babel` is expecting stage-2

Reply:
styled-components is just to make coding comfortable. It won't make any other problems

Reply:
I pointed this out, because you included the dependency here but was not using it anywhere in the code here. But I later saw that your other PRs were using this.

Reply:
Yes. My mistake. It happened because I added them to master first. Then created separate branches